content/spin/v2/python-components.md

```diff
@@ -74,12 +84,67 @@ A new Python component can be created using the following command:
 $ spin new -t http-py hello-world --accept-defaults
 ```
 
-This creates a directory of the following structure:
+### System Housekeeping (Use a Virtual Environment)
+
+Once the component is created, we can change into the `hello-world` directory, create and activate a virtual environment and then install the component's requirements:
+
+<!-- @selectiveCpy -->
+
+```console
+$ cd hello-world
+```
+
+Install the `virtualenv` tool:
```

Comment:
[venv](https://docs.python.org/3/library/venv.html) has been built-in to Python since 3.3, so there's no need to install `virtualenv` anymore unless you need more advanced functionality than `venv` provides, which does not appear to be the case here (i.e. we're only using `python3 -m venv`, not `python3 -m virtualenv`).

``

Reply:
Thanks, this is fixed in the next commit/push.